pynest/nest/lib/hl_api_simulation.py

```diff
@@ -190,99 +190,134 @@ def SetKernelStatus(params):
 
     Other parameters
     ----------------
-    The following parameters can be written to the kernel status dictionary.
+    The following parameters are available from the kernel status dictionary.
+    This is a list of all kernel parameters, the keys flagged with
+    `(read only)` are internally calculated and cannot be user defined.
```

Comment:
internally calculated -> set by the NEST kernel

Reply:
done